### PR TITLE
Added OS Client Binary and README

### PR DESCRIPTION
**Pull request by ericm**
Related issue: #95

### CODE REVIEW

File: gnoi_os/gnoi_os.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"google.golang.org/grpc"
+)
+
+var (
+	targetAddr = flag.String("target_addr", ":9339", "The target address in the format of host:port")
+	targetName = flag.String("target_name", "", "The target name used to verify the hostname returned by TLS handshake")
+	version    = flag.String("version", "", "Version of the OS required when using the activate operation")
+	osFile     = flag.String("os", "", "Path to the OS image for the install operation")
+	op         = flag.String("op", "", "OS service operation. Can be one of: install, activate, verify")
+	timeOut    = flag.Duration("time_out", 5*time.Second, "Timeout for the operation, 5 seconds by default")
```
Comment (samribeiro) [inline gnoi_os/gnoi_os.go:35]: just an FYI that we may need to test that this timeout works for installing large OS images.

Comment (ericm) [reply gnoi_os/gnoi_os.go:35]: Thats true

File: gnoi_os/gnoi_os.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	// TODO: Add Install RPC call
+}
+
+// activate the OS version to be used upon next reboot on the target.
+func activate() {
+	if *version == "" {
+		log.Exit("No version provided. Provide one with -version")
+	}
+	err := client.Activate(ctx, *version)
+	if err != nil {
```
Comment (loriscn12) [inline gnoi_os/gnoi_os.go:88]: save one line.

File: gnoi_os/gnoi_os.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+		log.Exit("Failed Activate:", err)
+	}
+}
+
+// verify the version of the OS running on the target.
```
Comment (loriscn12) [inline gnoi_os/gnoi_os.go:93]: verify verifies etc...

fix all the occurrences also.

**Comment by ericm** [comment]
Tidied up :)

**Pull request merged**